department-of-veterans-affairs/va.gov-cms
# VACMS-7525 update paragraph entity browser styles

## Description

Closes #7525. Fixes modal header color, spacing issues inside of paragraph browser

## Testing done
Check paragraph entity browser, block browser, etc. Modals have the VA blue header color now. Spacing issues fixed for paragraph browser. Filter actions on the same line.

## Screenshots
Before:
<img width="1063" alt="Screen Shot 2022-01-07 at 10 25 47 AM" src="https://user-images.githubusercontent.com/11279744/148615007-f07d4553-b147-4add-921c-356a09e396ee.png">
<img width="1666" alt="Screen Shot 2022-01-07 at 1 14 19 PM" src="https://user-images.githubusercontent.com/11279744/148615021-f3e9f115-977b-4b02-87af-db2e73d6f2ab.png">

After:
<img width="1118" alt="Screen Shot 2022-01-07 at 1 29 43 PM" src="https://user-images.githubusercontent.com/11279744/148615045-3109c1c2-79ab-4c32-a3ea-de5348f7b24b.png">
<img width="1146" alt="Screen Shot 2022-01-07 at 1 31 02 PM" src="https://user-images.githubusercontent.com/11279744/148615060-ef99384a-91fc-47e5-b98f-072c2869c653.png">



## QA steps

As any user with editor powers, add paragraph to a "content block" field (e.g. as seen on centralized content node type), verify that the paragraph browser that pops up is visually improved.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
